src/GeneralStateTestsFiller/stEWASMTests/callFiller.yml

```diff
@@ -40,22 +40,28 @@ call:
           (import "ethereum" "getBalance"  (func $getBalance (param i32 i32)))
           (import "ethereum" "storageStore" (func $storageStore (param i32 i32)))
           (import "ethereum" "getAddress" (func $getAddress (param i32)))
+          (import "debug" "printMemHex" (func $printMemHex (param i32 i32)))
           (memory 1 )
           (data (i32.const 0)  "\01\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\ef\be\ad\de")
           (data (i32.const 20) "\01\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00")
           (data (i32.const 52) "\42\04\20\42\04\20\42\00")
           (export "memory" (memory 0))
           (export "main" (func $main))
           (func $main
+            (local $result i32)
             ;; Memory layout:
             ;;   0 -  20 bytes: address (4)
             ;;  20 -  52 bytes: value (0)
             ;;  52 -  56 bytes: data (0x42004200)
             ;;  56 -  60 bytes: result
-            (i32.store (i32.const 56) ;; store the result so it is not returned
-            ;;          gas              addrOffset    valOffset      dataOffset     dataLength
+            (set_local $result 
+              ;;          gas                addrOffset    valOffset      dataOffset     dataLength
               (call $call (i64.const 200000) (i32.const 0) (i32.const 20) (i32.const 52) (i32.const 4)))
-            ;;drop
+
+            (i32.store (i32.const 132) (get_local $result))
+
+            (call $printMemHex (i32.const 132) (i32.const 4))
```

Comment:
debug statements causes problems when hera is built without the -DHERA_DEBUGGING=ON flag, debug statements has been removed from all the other tests.